package CfgBag;

message Grid {
	required uint32 type = 1;
	required uint32 num = 2;
	required uint32 cost = 3;
}

message Grids {
	repeated Grid grids = 1;
}